app/frontend/src/components/common/TeamTable.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <v-container>
+    <!-- search input -->
+    <div class="team-search mt-6 mt-sm-0">
+      <v-text-field
+        v-model="search"
+        append-icon="mdi-magnify"
+        label="Search"
+        single-line
+        hide-details
+        class="pb-5"
+      />
+    </div>
+    <!-- table alert -->
+    <v-alert v-if="alertShow" :type="alertType" tile dense>{{ alertMessage }}</v-alert>
+    <!-- table header -->
+    <v-data-table
+      class="team-table"
+      :headers="headers"
+      item-key="id"
+      :items="users"
+      must-sort
+      :search="search"
+      :loading="loading"
+      loading-text="Loading... Please wait"
+      :options="options"
+    >
+      <template v-slot:item.username="{ item }">
+        <v-tooltip bottom>
+          <span>{{ item.id }}</span>
+          <template v-slot:activator="{ on }">
+            <span v-on="on">{{ item.username }}</span>
+          </template>
+        </v-tooltip>
+      </template>
+      <template v-slot:item.newRole>Dropdown TBD</template>
+    </v-data-table>
+  </v-container>
+</template>
+
+<script>
+import commonFormService from '@/services/commonFormService';
+
+export default {
+  name: 'TeamTable',
+  data: () => ({
+    headers: [
+      { text: 'Username', align: 'start', value: 'username' },
+      { text: 'Name', align: 'start', value: 'fullname' },
+      { text: 'Email', align: 'start', value: 'email' },
+      { text: 'Current Role', align: 'start', value: 'currentRole' },
+      { text: 'New Role', align: 'end', value: 'newRole', sortable: false }
```

Comment:
just a thought. i'm assuming you'll have a dropdown to change the role instead of the last two colums. you might need add `filterable: false` to newRole to make it not searchable too

Reply:
or maybe we dont even need the search box

Reply:
I'll go ahead and add the `filterable: false` as we don't want to sort on that. However, I suspect it will still be useful to keep the search box in as there may be a case where a form has a ton of users, and we want to enable an administrator to be able to quickly find a specific user they had in mind - that or search filter down to only a specific role as another use case.